packages/teleterm/src/ui/ClusterLogout/useClusterLogout.ts

```diff
@@ -0,0 +1,45 @@
+import { useAppContext } from '../appContextProvider';
+import useAsync from '../useAsync';
+import { useEffect } from 'react';
+
+export function useClusterLogout({ clusterUri, onClose, clusterTitle }: Props) {
+  const ctx = useAppContext();
+  const [{ status, statusText }, removeCluster] = useAsync(async () => {
+    //TODO gzdunek: logout and removeCluster should be combined into a single acton in tshd
+    await ctx.clustersService.logout(clusterUri);
+    await ctx.clustersService.removeCluster(clusterUri);
```

Comment:
I was going to ask "What about error handling?", but I see it already works quite nicely. 👍